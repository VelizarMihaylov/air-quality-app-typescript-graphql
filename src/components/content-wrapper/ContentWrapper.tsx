import React from 'react'
import styled from 'styled-components'

type ContentWrapperProps = {
  children?: React.ReactNode
}

const StyledContentWrapper = styled.div`
  margin: 0 auto;
  max-width: ${({ theme }) => theme.spacing(60)};
`

export const ContentWrapper: React.FC<ContentWrapperProps> = ({ children }) => {
  return <StyledContentWrapper>{children}</StyledContentWrapper>
}
