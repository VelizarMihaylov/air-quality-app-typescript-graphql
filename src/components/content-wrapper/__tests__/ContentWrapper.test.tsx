import React from 'react'
import { create } from 'react-test-renderer'

import { ContentWrapper } from '../ContentWrapper'

import { ThemeProvider } from 'styled-components'
import generateTheme from 'theme'

describe('ContentWrapper', () => {
  it('should render without throwing an error', () => {
    const renderer = create(
      <ThemeProvider theme={generateTheme()}>
        <ContentWrapper />
      </ThemeProvider>
    )

    expect(renderer.toJSON()).toMatchSnapshot()
  })
})
