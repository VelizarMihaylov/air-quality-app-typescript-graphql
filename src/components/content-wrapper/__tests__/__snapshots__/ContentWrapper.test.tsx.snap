// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentWrapper should render without throwing an error 1`] = `
<div
  className="sc-bdfBwQ gTLSEq"
/>
`;
